### DISCUSSION TITLE
模块化

### DISCUSSION THREAD

**Discussion by terenceyhj** [Huckies/qmk_port_ch582#21] [Ideas]
Hi.O-H-M2
 我有几点不成熟的想法！
 1，能否加超轻量级的操作系统（如[eventos-basic](https://gitee.com/event-os/eventos-basic)，RAM占用小于1k）
 2，指纹功能（主要针对需登录认证的应用，进行认证登录）
 3，软件功能模块化

**Comment by Huckies** [comment]
Hi, terenceyhj
1.请问加入非抢占式操作系统的意义是？
2.指纹功能在物色相关硬件后可以轻易添加
3.我认为目前已经是「模块化」了，RGB等附加功能可以通过编译规则单独启用/关闭

**Comment by OctopusZ** [comment]
加入指纹的目的是什么？
如果是只是为了Windows hello
那使用专门的设备我觉得是一个更好的选择
